tokenization/permutate.test.js

```diff
@@ -3,14 +3,25 @@ const permutate = require('./permutate')
 
 module.exports.tests = {}
 
+function generatePhrase (...spans) {
```

Comment:
This is very similar to https://github.com/pelias/parser/blob/master/tokenization/split.js#L37-L41, I wonder if it would be possible to share that code somehow?

Reply:
Hum, yes we can add it in Span as a global function ? (to be used as `Span.connectSiblings`  or `require('./Span').connectSiblings` for example ?)

https://github.com/pelias/parser/blob/cd87ea401a75dc493fd3ac20e8429548a431ce1b/tokenization/Span.js#L73

Reply:
Done in 519d117